src/lib/components/License/LicenseResults.js

```diff
@@ -40,6 +40,7 @@ export const LicenseResults = withState(
                 >
                   <Radio
                     checked={_get(values, 'selectedLicense.title') === title}
+                    className={!description && 'mt-0'}
```

Comment:
this will create an error in the console since you will pass boolean instead of expected string to the classname prop

Reply:
Good catch, thanks!